### PR TITLE
Helm based Prometheus and grafana addon installation for meshops v1

### PR DESCRIPTION
**Pull request by Revolyssup**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Revolyssup will you address these failed static checks?

**Comment by leecalcote** [comment]
So close…

**Comment by Revolyssup** [comment]
> So close…

all green

File: templates/patches/service-loadbalancer.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "spec": {
+      "type": "LoadBalancer"
+    }
+  }
```
Comment (utkarsh-pro) [inline templates/patches/service-loadbalancer.json:5]: @Revolyssup, a horrible nitpick - indentation

**Review by utkarsh-pro — APPROVED** [review]
LGTM

**Comment by utkarsh-pro** [comment]
All green and approved - lets merge?

**Pull request merged**